contracts/citizens/DeveloperSkillLevels.sol

```diff
@@ -98,5 +104,19 @@ contract DeveloperSkillLevels {
         skillLevelAverages[developer] = newSkillLevelAverage;
         skillLevelRatings[developer][msg.sender] = rating;
         emit Rated(developer, rating, msg.sender);
+
+        eas.attest(
+            AttestationRequest({
+                schema: easSchemaUID,
+                data: AttestationRequestData({
+                    recipient: developer,
+                    expirationTime: NO_EXPIRATION_TIME,
+                    revocable: false,
+                    refUID: EMPTY_UID,
+                    data: abi.encode(0), // TO DO
+                    value: 0
+                })
+            })
+        );
```

Comment:
This is really cool. Are multiple attestations ok all coming from one place, or will `data` handle that for us?

Reply:
@johnmark13 In the 'data', we can pass the role name and the rating, with a different parameter value per role:
- `data: abi.encode("Developer", rating),`
- `data: abi.encode("Operator", rating),`
- `data: abi.encode("Marketeer", rating),`